From c63745a2777648d530898bae5af9d25266472b05 Mon Sep 17 00:00:00 2001
From: Samuel Holland <samuel@sholland.org>
Date: Sat, 21 Dec 2019 17:00:40 +0100
Subject: [PATCH 08/15] Ignore broken and non-applicable tests

c-link-to-rust-va-list-fn: unstable feature, broken on aarch64, #56475
env-funky-keys: can't handle LD_PRELOAD (e.g. sandbox)
long-linker-command-lines: takes >10 minutes to run (but still passes)
simd-intrinsic-generic-bitmask.rs: broken on BE, #59356
simd-intrinsic-generic-select.rs: broken on BE, #59356
sparc-struct-abi: no sparc target
sysroot-crates-are-unstable: can't run rustc without RPATH
---
 src/test/codegen/sparc-struct-abi.rs                            | 1 +
 src/test/run-make-fulldeps/c-link-to-rust-va-list-fn/Makefile   | 2 ++
 src/test/run-make-fulldeps/long-linker-command-lines/Makefile   | 2 ++
 src/test/run-make-fulldeps/sysroot-crates-are-unstable/Makefile | 2 ++
 src/test/ui/env-funky-keys.rs                                   | 1 +
 src/test/ui/simd/simd-intrinsic-generic-bitmask.rs              | 2 ++
 6 files changed, 10 insertions(+)

diff --git a/src/test/codegen/sparc-struct-abi.rs b/src/test/codegen/sparc-struct-abi.rs
index 78e5b14a..6f93e932 100644
--- a/src/test/codegen/sparc-struct-abi.rs
+++ b/src/test/codegen/sparc-struct-abi.rs
@@ -4,6 +4,7 @@
 
 // only-sparc64
 // compile-flags: -O --target=sparc64-unknown-linux-gnu --crate-type=rlib
+// ignore-test
 #![feature(no_core, lang_items)]
 #![no_core]
 
diff --git a/src/test/run-make-fulldeps/c-link-to-rust-va-list-fn/Makefile b/src/test/run-make-fulldeps/c-link-to-rust-va-list-fn/Makefile
index f124ca2a..363b18f0 100644
--- a/src/test/run-make-fulldeps/c-link-to-rust-va-list-fn/Makefile
+++ b/src/test/run-make-fulldeps/c-link-to-rust-va-list-fn/Makefile
@@ -1,3 +1,5 @@
+# ignore-aarch64
+
 -include ../tools.mk
 
 all:
diff --git a/src/test/run-make-fulldeps/long-linker-command-lines/Makefile b/src/test/run-make-fulldeps/long-linker-command-lines/Makefile
index 5876fbc9..5f167ece 100644
--- a/src/test/run-make-fulldeps/long-linker-command-lines/Makefile
+++ b/src/test/run-make-fulldeps/long-linker-command-lines/Makefile
@@ -1,3 +1,5 @@
+# ignore-test
+
 -include ../tools.mk
 
 all:
diff --git a/src/test/run-make-fulldeps/sysroot-crates-are-unstable/Makefile b/src/test/run-make-fulldeps/sysroot-crates-are-unstable/Makefile
index 9e770706..6d92ec5c 100644
--- a/src/test/run-make-fulldeps/sysroot-crates-are-unstable/Makefile
+++ b/src/test/run-make-fulldeps/sysroot-crates-are-unstable/Makefile
@@ -1,3 +1,5 @@
+# ignore-test
+
 -include ../tools.mk
 
 all:
diff --git a/src/test/ui/env-funky-keys.rs b/src/test/ui/env-funky-keys.rs
index c5c824ac..f3fe047a 100644
--- a/src/test/ui/env-funky-keys.rs
+++ b/src/test/ui/env-funky-keys.rs
@@ -1,6 +1,7 @@
 // run-pass
 // Ignore this test on Android, because it segfaults there.
 
+// ignore-test
 // ignore-android
 // ignore-windows
 // ignore-cloudabi no execve
2.26.2

